### PR TITLE
support for safari

### PR DESCRIPTION
**Pull request by xdumaine**

### CODE REVIEW

**Comment by austinkelleher** [comment]
Thanks. I look forward to troubleshooting webrtc on Safari.

**Comment by charlieduong94** [comment]
+1 Thanks for this @xdumaine

**Comment by wenkesj** [comment]
+1 Fits my use-case perfectly

File: src/diagnostics/VideoBandwidthTest.js [hunk 1]
```diff
@@ -199,9 +199,14 @@ export default class VideoBandwidthTest extends Test {
   completed () {
     const isWebkit = 'WebkitAppearance' in document.documentElement.style;
     const isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
-    this.call.pc1.getLocalStreams()[0].getTracks().forEach((track) => {
-      track.stop();
-    });
+    window.thecall = this.call;
```
Comment (hemp) [inline src/diagnostics/VideoBandwidthTest.js]: can 🔫 this i think

**Comment by xdumaine** [comment]
Oops. Meant to get that. I wish safari debug tools were better.

**Pull request merged**